packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Conventions:
+ * _message: is for unformatted text that will be shown as-is to
+ * the user.
+ * _text: is for text that will appear in the UI, possibly with
+ * decorations, e.g., $(x) uses the https://octicons.github.com/ and should not
+ * be localized
+ *
+ * If ommitted, we will assume _message.
+ */
+export const messages = {
+  salesforcedx_vscode_core_not_installed_text:
+    'salesforce.salesforcedx-vscode-lwc failed to activate. Ensure that you have the latest version of salesforce.salesforcedx-vscode-core installed and activated',
+
+  force_lightning_lwc_create_text: 'SFDX: Create LWC Bundle',
+
+  warning_prompt_lightning_bundle_overwrite:
+    'An LWC bundle with the specified path already exists in your workspace. Do you want to overwrite any existing files in this bundle?',
+  warning_prompt_yes: 'Yes',
+  warning_prompt_no: 'No'
```

Comment:
Can we change these from 'Yes' and 'No' to 'Overwrite' and 'Cancel'? We try to be as specific as possible about these confirmations, since people often just skim the text and don't necessarily realize what they're agreeing to.

Reply:
Yes. The text was taken from the other scaffolding commands. I'll make those changes over there as well. Specifically, I will create a new`confirm_overwrite_text`